index.js

```diff
@@ -187,6 +187,9 @@ function use (extensions) {
     this.on('close', onfinalize)
     this.on('end', onfinalize)
     this.on('finish', this.finalize)
+    this.on('error', function (err) {
+      debug('stream error', err)
+    })
```

Comment:
I wouldn't assume DEBUG=something being passed means the executor is watching the output, I have deployed with DEBUG set for retroactive inspection.